src/pages/auth/registration/const.tsx

```diff
@@ -1,14 +1,12 @@
-import * as yup from 'yup'
-
 import { UserRegistrationData } from 'types'
 
-export const RegistrationFileds = ['name', 'email', 'password']
+enum FieldsEnum {
+  EMAIL = 'email',
+  NAME = 'name',
+  PASSWORD = 'password'
+}
```

Comment:
could you use `InputTypes` here?

Reply:
I can`t because it will be broken tipization, Input Types hasn`t **NAME**